Secrets/KeysExample.swift

```diff
@@ -1,6 +1 @@
-struct Keys {
-    struct Parse {
-        static let applicationId = ""
-        static let clientKey = ""
-    }
-}
+let x = 1
```

Comment:
Name Format Violation (Medium Severity): Variable name should be between 3 and 40 characters in length: 'x'